main.c

```diff
@@ -550,6 +582,9 @@ LRESULT CALLBACK WindowProcedure (HWND hwnd, UINT message, WPARAM wParam, LPARAM
         SendMessage(hwnd, WM_CLOSE, 0, 0);
         break;
       }
+      /* A timer to periodically tend to persistent connections */
+      SetTimer(hwnd, 0, 10000, ManagePersistent);
```

Comment:
Why do we wait 10 seconds before the first attempt to attach to daemons?

Reply:
Ah I know why. We don't wait 10 seconds - on startup we start all connections with `auto_connect` flag, this also handles persistent connections. This code gives us a chance to reattach.

Reply:
Yes, as you figured. But, on further thought, logically it may be better to do even the first start here in ManagePersistent but for now this works.